apps/laboratory/tests/wallet.spec.ts

```diff
@@ -39,8 +39,25 @@ sampleWalletTest.afterAll(async () => {
 })
 
 // -- Tests --------------------------------------------------------------------
+sampleWalletTest.only(
+  'it should show testnet and devnet disabled on solana',
+  async ({ library }) => {
+    if (library !== 'solana') {
+      return
+    }
+
+    await modalPage.openModal()
+    await modalPage.openNetworks()
+    await modalValidator.expectNetworksDisabled('Solana Testnet')
```

Comment:
Since we don't change something in the lab examples, why it's expected to be disabled?

Reply:
Because since the wallet does not support it, it's not on namespace and should not be enabled